src/app/pages/forms-and-validation/in-a-list/in-a-list.component.html

```diff
@@ -0,0 +1,36 @@
+<mat-toolbar color="primary" class="app-bar mat-elevation-z4" data-cy="pxb-toolbar">
+    <button data-cy="toolbar-menu" *ngIf="isSmall" style="margin-left: -8px" mat-icon-button (click)="toggleMenu()">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <div class="mat-title" [style.marginLeft.px]="isSmall ? 24 : 0">In a List</div>
+</mat-toolbar>
+<section class="mat-typography">
```

Comment:
I'd like to see a different class name here since "mat-typography" is not descriptive of what this section is. I've seen this in a few places, probably just a copy/paste issue. Let's keep an eye out for this in the future so we don't keep adding this class